components/expenses/ExpenseItemForm.js

```diff
@@ -39,6 +38,15 @@ import { Tooltip, TooltipContent, TooltipTrigger } from '../ui/Tooltip';
 import { ExpenseAccountingCategoryPill } from './ExpenseAccountingCategoryPill';
 import { ExpenseItemDescriptionHint } from './ItemDescriptionHint';
 
+function isURL(value) {
+  try {
+    new URL(value);
+    return true;
+  } catch (e) {
+    return false;
+  }
+}
+
```

Comment:
Maybe add it in https://github.com/opencollective/opencollective-frontend/blob/main/lib/utils.ts#L30?